.circleci/config.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+jobs:
+  build:
+    docker:
+      # CircleCI Go images available at: https://hub.docker.com/r/circleci/golang/
+      - image: circleci/golang:1.10-stretch
+
+    working_directory: /go/src/github.com/giantswarm/gsclientgen
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - v1-pkg-cache
+
+      - run: go get -u github.com/golang/dep/cmd/dep
+
+      - run: make build
+
+      - save_cache:
```

Comment:
We prefer not using circle-ci cache as we like to have reproducible builds.
I know this make builds a lot slower but in the other hand it guarantees that we deliver working software from scratch.

Reply:
Removed